src/base/base.scss

```diff
@@ -96,8 +96,6 @@ button,
 [role='button'],
 input[type='radio'],
 input[type='checkbox'] {
-  transition: 80ms cubic-bezier(0.33, 1, 0.68, 1);
-  transition-property: color, background-color, box-shadow, border-color;
```

Comment:
This might fix a bunch of the transition bugs we've seen? 🤩

Reply:
I _think_ so.. I feel a little worried deleting this but its probably fine? Just a transition 😅

Reply:
I don't believe there was functionality relying on this